tests/unit/test_bigquery_adapter.py

```diff
@@ -473,8 +473,17 @@ def test_replace(self):
         relation = BigQueryRelation.from_dict(kwargs)
         info_schema = relation.information_schema()
 
-        tables_schema = info_schema.replace(information_schema_view='__TABLES__')
-        assert tables_schema.information_schema_view == '__TABLES__'
+        tables_schema = info_schema.replace(information_schema_view='INFORMATION_SCHEMA.TABLES')
```

Comment:
`TABLE_STORAGE` is also needed for multi-region BigQuery setups as illustrated in this macro workaround this PR is solving for natively: https://github.com/GeneralMills/gmi_common_dbt_utils/blob/main/macros/bq_catalog.sql#L51